.github/workflows/spellcheck.yml

```
@@ -0,0 +1,20 @@
+name: Spellcheck
```

Hmm... could we rename this to something like "Style"? I'd like to add some other checks in here too, later.
```suggestion
name: Style
```